app/scripts/modules/netflix/fastProperties/domain/propertyStrategy.domain.ts

```diff
@@ -0,0 +1,82 @@
+
+
+import {PropertyCommand} from './propertyCommand.model';
+import {IStage} from 'core/domain/IStage';
+import {ManualJudgementStage} from './manualJudgementStage';
+import {IUser} from 'core/authentication/authentication.service';
+import {AcaTaskStage} from './acaTaskStage';
+import {AcaTaskStageConfigDetails} from './acaTaskStageConfigDetails.model';
+
+export interface PropertyStrategy {
+  name: string;
+  displayName: string;
+  description: string;
+  configDetails: any;
+  form: string;
+
+  isForcePush(): boolean;
+  isManual(): boolean;
+  isAca(): boolean;
+  buildStage(user: IUser, command: PropertyCommand, previousStage?: IStage): IStage;
+}
+
+export class ManualStrategy implements PropertyStrategy {
+  name: string = 'ManualStrategy';
+  displayName = 'Manual Judgement Strategy';
+  description: string = `Manual Strategy will add a Manual Judgement Stage after each Property Stage`;
+  configDetails: any = {};
+  form: string;
+
+  buildStage(user: IUser, command: PropertyCommand, previousStage?: IStage): IStage {
+    return new ManualJudgementStage(user, command, previousStage);
+  }
+  isForcePush() { return false; }
+  isManual() { return true; }
+  isAca() { return false; }
+
+  constructor(formName: string)  {
+    this.form = formName;
+  }
+
+}
+
+export class AcaStrategy implements PropertyStrategy {
+  name: string = 'AcaStrategy';
```

Comment:
Prefer explicit access modifiers, even though TS implicitly sets `public`.  Explicit access modifiers are required for NG2.